flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/reader/ReaderUtil.java

```diff
@@ -122,4 +123,24 @@ public static CombinedScanTask createCombinedScanTask(
 
     return new BaseCombinedScanTask(fileTasks);
   }
+
+  public static CombinedScanTask createCombinedScanTask(
```

Comment:
can you elaborate why this change is needed?

Reply:
The other `createCombinedScanTask` (described above in this class) is still used in other test classes which include JUnit4. This method still takes the `TemporaryFolder` argument. So, for this commit, I choose the method overloading. Once the other classes are migrated into JUnit5, I will remove the old `createCombinedScanTask` (which takes the `TemporaryFolder` argument). I will add a comment line for this.